From 404ffe5679e6430d37282d5eb258993cfda8dae4 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Fri, 12 Jan 2018 14:33:11 -0800
Subject: [PATCH] TRITON-49 vms.changefeed.test.js throwing error makes whole
 VMAPI tests suite abort Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
 Approved by: Josh Wilsdon <jwilsdon@joyent.com>

---
 test/vms.changefeed.test.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/vms.changefeed.test.js b/test/vms.changefeed.test.js
index dbf688c..179c6fc 100644
--- a/test/vms.changefeed.test.js
+++ b/test/vms.changefeed.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
@@ -97,7 +97,7 @@ function waitForValue(url, key, value, callback) {
             times++;
 
             if (times === TIMEOUT) {
-                throw new Error('Timeout waiting on ' + url);
+                callback(new Error('Timeout waiting on ' + url));
             } else {
                 setTimeout(function () {
                     waitForValue(url, key, value, callback);
-- 
2.21.0

